### PR TITLE
feature: emit ftx orderbook snapshot

### PR DESCRIPTION
**Pull request by ycdesu**
The orderbook snapshot is published to the standard stream. Now you could print orderbook snapshot of a specific market by `go run ./cmd/bbgo orderbook --session=ftx --symbol=btc/usdt`.

Note that I haven't implemented the checksum mechanism now. This will be a good improvement and may be contributed by someone who is interested.

screenshot:

image

### CODE REVIEW

**Comment by c9s** [comment]
Awesome! can you make the command compatible with binance and max? meaning, use the symbol name in uppercase , thanks!

File: pkg/cmd/orderbook.go [hunk 1]
```diff
@@ -29,13 +30,19 @@ var orderbookCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("can't get the symbol from flags: %w", err)
 		}
+		if symbol == "" {
+			return fmt.Errorf("symbol is not found")
+		}
 
 		s := ex.NewStream()
 		s.Subscribe(types.BookChannel, symbol, types.SubscribeOptions{})
+		s.OnBookSnapshot(func(book types.OrderBook) {
```
Comment (c9s) [inline pkg/cmd/orderbook.go:39]: you can use environment to load exchange session

Comment (ycdesu) [reply pkg/cmd/orderbook.go:39]: @c9s Could you give me an example?

**Comment by ycdesu** [comment]
> Awesome! can you make the command compatible with binance and max? meaning, use the symbol name in uppercase , thanks!

I've modified the comment. In the underlying implementation, I also convert the input symbol to uppercase string.

File: pkg/cmd/orderbook.go (hunk 1 above)
Comment (c9s) [reply pkg/cmd/orderbook.go:39]: oh but that way you will need to fulfill the implementation

**Review by c9s — APPROVED** [review]

**Pull request merged**